test/cypress/e2e/integration/app-frontend/likert.js

```diff
@@ -0,0 +1,8 @@
+/// <reference types='cypress' />
+/// <reference types="../../support" />
+
+describe('Likert', () => {
+  it('Fill out required likert', () => {
+    cy.completeTask4Form();
```

Comment:
I guess Cypress would fail if it couldn't complete the form - but maybe there should be some assertions here as well? I see you added a summary component to the layout, those could work for double-checking the correct values get set.

Reply:
I was planning on keeping this pr simple and extend the likert tests in a seperate one. The Summary component is currentlig not showing correct values, this pr fikses this issue https://github.com/Altinn/app-frontend-react/pull/478

Reply:
Ah, I see! That makes sense.

Reply:
The summary component looks like this : 
![image](https://user-images.githubusercontent.com/43168787/191432515-9a49c127-4440-4b0d-9be8-8cafd5583738.png)